Bottom: 372db6d6f38932908d2158a9db9fc64acf584a23
Top:    f416829eb74479296675d93b02a6e7924e88fa0e
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-03 21:14:33 -0700

fixup-i-dont-know-where


---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
index a5f7fe9ebf2..ddef79a5b68 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
@@ -552,7 +552,6 @@ class FileHandleTestCase(lldbtest.TestBase):
 
     @add_test_categories(['pyapi'])
     @skipIf(py_version=['<', (3,)])
-    @skipIf(True) # fixme multiple problems with this
     def test_string_out(self):
         f = io.StringIO()
         status = self.debugger.SetOutputFile(f)
